android/walletlib/src/main/java/com/solana/mobilewalletadapter/walletlib/protocol/MobileWalletAdapterConfig.java

```diff
@@ -13,6 +13,9 @@ public class MobileWalletAdapterConfig {
 
     public final boolean supportsSignAndSendTransactions;
 
+    @IntRange(from = 0)
+    public final long noConnectionWarningTimeoutMillis;
```

Comment:
NIT: elsewhere in `walletlib` and `clientlib` we use the suffix `Ms`, rather than `Millis`